build.gradle

```diff
@@ -735,3 +736,88 @@ task distRpm(dependsOn: "jpackageImage", type: Rpm) {
 	prefix '/opt' // Relocations: /opt
 	arch = 'x86_64'
 }
+
+task dockerPullOracleLinux8(type: DockerPullImage) {
+	group = "RH8 RPM build container"
+	image = "openjdk:17-oraclelinux8"
+	description = "Pulls a docker image for oraclelinux8 from ${image}."
+}
+
+task dockerCreateOracleLinux8(dependsOn: dockerPullOracleLinux8, type: DockerCreateContainer) {
+	group = "RH8 RPM build container"
+	description = "Creates an OracleLinux8 container named ${oracleLinux8ContainerName}."
+	imageId = dockerPullOracleLinux8.getImage()
+	containerName = oracleLinux8ContainerName
+	hostConfig.binds = ["${projectDir}":"/source/anet"]
+	stdinOpen = true
+}
+
+task dockerStartOracleLinux8(type: DockerStartContainer) {
+	group = "RH8 RPM build container"
+	description = "Starts ${oracleLinux8ContainerName} container."
+	containerId = oracleLinux8ContainerName
+}
+
+task dockerStopOracleLinux8(type: DockerStopContainer) {
+	group = "RH8 RPM build container"
+	description = "Stops ${oracleLinux8ContainerName} container."
+	containerId = oracleLinux8ContainerName
+}
+
+task dockerRemoveOracleLinux8(type: DockerRemoveContainer) {
+	group = "RH8 RPM build container"
+	description = "Removes ${oracleLinux8ContainerName} container."
+	containerId = oracleLinux8ContainerName
+}
+
+task dockerInstallYumOracleLinux8(type: DockerExecContainer) {
```

Comment:
Many of these tasks don't make sense on their own, they're merely steps in a process. Why not add a script with all the individual steps, e.g. `scripts/create-rpm.sh`, copy that to the container and run it there?

Reply:
We should discuss this. Because I was going to do the same thing but it was suggested to do it with gradle task. Let's decide and apply accordingly.

Reply:
That building the RPM should be a Gradle task does not necessarily mean that each individual step in the build process must be a separate Gradle task…

Reply:
then our decision is to do a single gradle task and move the other steps into script.  Can you talk that with Maarten.  Because I suggested the same solution